a2ml/api/utils/__init__.py

```diff
@@ -0,0 +1,10 @@
+from collections.abc import Iterable
+
+def to_list(obj):
+    if obj:
+        if isinstance(obj, Iterable) and not isinstance(obj, str):
+            return list(obj)
+        else:
+            return [obj]
+    else:
+        return []
```

Comment:
I couldn't find analogy for ruby `Array()` pattern:

```ruby
Array(nil) = []
Array(1) = [1]
Array([1]) = [1]
Array([1,2,3]) = [1,2,3]
```

maybe there is some build-in function for it